recovery/src/main/java/com/kin/ecosystem/recovery/widget/PasswordEditText.java

```diff
@@ -94,7 +95,7 @@ private void setupPasswordField(boolean addRevealIcon, String hint) {
 		passwordField.setTypeface(Typeface.SANS_SERIF);
 		passwordField.setTextColor(colorGray);
 		passwordField.setHintTextColor(colorGray);
-		passwordField.setTextSize(TypedValue.COMPLEX_UNIT_SP, 14f);
+		passwordField.setTextSize(TypedValue.COMPLEX_UNIT_PX, textSize);
```

Comment:
why changing to PX instead of SP?

Reply:
because textsize comes as px from resources.